src/pages/HomePage.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import "./HomePage.css";
+import PropTypes from "prop-types";
+import Grid from "@material-ui/core/Grid";
+
+const HomePage = ({ user }) => {
+  return (
+    <Grid container xs={12} sm={12} md={7} lg={6} xl={5} className="grid">
```

Comment:
Replace `className=“grid”` with `className=“HomePage”`, if you are wrapping up the CSS above.

Reply:
xs sm md lg xl attributes can only be placed in grid items.